index.html

```diff
@@ -209,6 +189,19 @@ <h2 class="outro2">LET'S CONNECT</h2>
         </div>
       </div>
     </section>
+  <section class="contactme">
+    <h1 class="contacttxt font">Contact me</h1>
+    <p class="paragraph font">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/meqwgnyz" method="post">
+    <input type="text" name="name" id="name" maxlength="30" placeholder="Enter your name" class="inputtxt1 radius" required>
+   
+    <input name="Email" id="email" type="email" placeholder="Enter your email address" class="inputtxt2 radius" required>
+  
+    <textarea name="textarea" id="message" placeholder="Write your message here" cols="30" rows="10" class="inputtxt3 radius" maxlength="500"></textarea>
```

Comment:
[OPTIONAL] All input fields have the `required` attribute to activate the validation on submit. Kindly add required to `textarea` too.

Reply:
Done!